### PR TITLE
Levelbuilder changes for non-EN videos

### PR DESCRIPTION
**Pull request by bethanyaconnor**
- Add locale column to video view.
- Add locale dropdown to video form viewer.
- Thumbnails should only be generated for English videos.
- Throw an exception if an English video of the same key doesn't exist when trying to add a non-EN video.

Screenshots:
screenshot 2019-02-22 at 3 21 04 pm
screenshot 2019-02-22 at 3 45 38 pm

### CODE REVIEW

File: dashboard/app/controllers/videos_controller.rb [hunk 1]
```diff
@@ -44,6 +44,12 @@ def create
     filename = upload_to_s3
     @video = Video.new(video_params.merge(download: "https://videos.code.org/#{filename}"))
 
+    if @video.locale != 'en-US'
+      unless Video.exists?(key: @video.key, locale: 'en-US')
```
Comment (Hamms) [inline dashboard/app/controllers/videos_controller.rb]: nit; can we use something like `i18n.default_locale`, both here and in the model?

**Review by Hamms — APPROVED** [review]
this looks great!

File: dashboard/app/helpers/locale_helper.rb [hunk 2]
```diff
@@ -34,6 +34,15 @@ def options_for_locale_select
     options
   end
 
+  def options_for_locale_code_select
+    options = []
+    Dashboard::Application::LOCALES.each do |locale, data|
+      next unless I18n.available_locales.include?(locale.to_sym) && data.is_a?(Hash)
```
Comment (joshlory) [inline dashboard/app/helpers/locale_helper.rb]: What are cases where `I18n.available_locales` doesn't include a given locale, or where the data isn't a hash?

**Review by joshlory — APPROVED** [review]

File: dashboard/app/controllers/videos_controller.rb (hunk 1 above)
Comment (bethanyaconnor) [reply dashboard/app/controllers/videos_controller.rb]: Done!

File: dashboard/app/helpers/locale_helper.rb (hunk 2 above)
Comment (bethanyaconnor) [reply dashboard/app/helpers/locale_helper.rb]: Available locales won't include [debug locales in production](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/application.rb#L94). If you looks at [locals.yml](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/locales.yml), half of the data isn't a hash. I copied most of this code from above but I'm now realizing that we can just iterate over the available_locales as we don't use anything in "data" for this function. I'll make that change.

**Pull request merged**